From: Thorsten Glaser <tg@mirbsd.org>
Subject: Declare package non-GNU.
--- a/configure.ac
+++ b/configure.ac
@@ -2,12 +2,11 @@ dnl Process this file with autoconf to p
 UPDATED='September 18, 2018'
 AC_INIT(LibDsk, 1.5.9)
 AC_CONFIG_SRCDIR(lib/dskopen.c)
-AC_CONFIG_AUX_DIR(config)
 AC_CONFIG_MACRO_DIR(m4)
 AC_CANONICAL_SYSTEM
 LT_INIT
 AM_CONFIG_HEADER(config.h)
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([foreign])
 
 dnl Support for the Java Native Interface bindings
 AC_ARG_WITH(jni,
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS= . include lib tools man doc
+SUBDIRS= . include lib tools man
 ACLOCAL_AMFLAGS=-I m4
 
 EXTRA_DIST= $(srcdir)/config/* \
